zeroize/derive/src/lib.rs

```diff
@@ -22,7 +22,7 @@ decl_derive!(
     /// Supports the following attributes:
     ///
     /// On the item level:
-    /// - `#[zeroize(drop)]`: call `zeroize()` when this item is dropped
+    /// - `#[zeroize(drop)]`: *deprecated* use `ZeroizeOnDrop` instead
```

Comment:
To properly deprecate this I think it'd be good to emit a `#[deprecated(...)]` attribute in the generated code, although I'm fine with addressing that in a follow-up PR.

Reply:
I am looking into it.